web/public/js/controllers/session-modal-controller.js

```diff
@@ -10,6 +10,7 @@
     $scope.applyForModel = applyForModel;
     $scope.currentUser = currentUser;
     $scope.referer = referer;
+    $scope.notes = "N/A";
```

Comment:
Need to add this to the [messages.po file](https://github.com/CoderDojo/cp-zen-platform/blob/master/web/locale/en_US/messages.po) and use $translate.instant e.g. [like this line](https://github.com/josmas/cp-zen-platform/blob/application_notes%23564/web/public/js/controllers/session-modal-controller.js#L76)